Dockerfile

```diff
@@ -41,7 +41,8 @@ RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master
  && ./get_helm.sh \
  && mv /usr/local/bin/helm /usr/local/bin/helm2
 
-RUN ./tools/install-jb.sh v0.4.0
+RUN ./tools/install-jb.sh v0.4.0 \
+ && ./tools/install-kustomize.sh 4.5.7 /usr/local/bin
```

Comment:
We may want to consider configuring Renovate for the Kustomize version (probably just with regex manager).

Reply:
Added a regex manager.